torch/ao/pruning/_experimental/pruner/base_structured_sparsifier.py

```diff
@@ -0,0 +1,74 @@
+from typing import Set, Type
+import torch
+from torch import nn
+from torch.nn.utils import parametrize
+
+from torch.ao.pruning import BaseSparsifier
+from .parametrization import FakeStructuredSparsity, BiasHook
+
+__all__ = ["BaseStructuredSparsifier"]
+
+SUPPORTED_STRUCTURED_PRUNING_MODULES = {  # added to config if None given
+    nn.Linear,
+    nn.Conv2d,
+}
+
+
+class BaseStructuredSparsifier(BaseSparsifier):
```

Comment:
Technically there is no such thing as "Structured Sparsity". Although most readers would understand that this means "structured pruning", if we want to be consistent, we should not use this name